src/quo2/components/wallet/transaction_progress/view.cljs

```diff
@@ -0,0 +1,306 @@
+                                                 "/4")
+    (= network-state :finalized)            "Epoch 181,329"
+    (and (= network-type :mainnet)
+         (= network-state :error))          "0/4"
+    (and (not= network-type :mainnet)
+         (or (= network-state :finalising)
+             (= network-state :confirmed))) "1/1"
+    (and (= network-type :arbitrum)
+         (= network-state :sending))        "1/1"
+    (not= network-type :mainnet)            "0/1"))
+
+(defn get-status-icon
+  [network-type network-state]
+  (cond
+    (and (= network-type :arbitrum)
+         (= network-state :sending))   ["positive-state"
+                                        (colors/theme-colors colors/white
+                                                             colors/success-50
+                                                             colors/success-60)]
+    (or (= network-state :pending)
+        (= network-state :sending))    ["pending-state"
+                                        (colors/theme-colors colors/white
+                                                             colors/neutral-50
+                                                             colors/neutral-60)]
+    (or (= network-state :confirmed)
+        (= network-state :finalising)) ["positive-state"
+                                        (colors/theme-colors colors/white
+                                                             colors/success-50
+                                                             colors/success-60)]
+    (= network-state :finalized)       ["diamond"
+                                        (colors/theme-colors colors/white
+                                                             colors/success-50
+                                                             colors/success-60)]
+    (= network-state :error)           ["negative-state"
+                                        (colors/theme-colors colors/danger
+                                                             colors/danger-50
+                                                             colors/danger-60)]))
+
+(defn transaction-progress
+  [{:keys [title
+           on-press
+           accessibility-label
+           network-type
+           network-state
+           start-interval-now
+           override-theme]}]
+  (let [count (reagent/atom 0)]
+    (rn/use-effect
+     (fn []
+       (when start-interval-now
+         (start-interval network-state))
+       (clear-counter)
+       (fn []
+         (stop-interval)))
+     [network-state])
+    [rn/view
+     [rn/touchable-without-feedback
+      {:on-press            on-press
+       :accessibility-label accessibility-label}
+      [rn/view
+       {:style style/box-style}
+       [rn/view
+        {:style style/title-item-container}
+        [rn/view
+         {:style style/inner-container}
+         [load-icon "placeholder" (colors/theme-colors colors/white colors/neutral-50 colors/neutral-60)]
```

Comment:
This call to `colors/theme-colors` doesn't look right. The third argument should be the theme you want to override, not another color.